definitions/ext-tinyproxy/definition.yml

```diff
@@ -0,0 +1,58 @@
+# -- Mandatory fields --
+domain: EXT
+type: TINYPROXY
+
+synthesis:
+  # -- Mandatory fields --
+  name: targetName
+  identifier: targetName
+
+  conditions:
+    - attribute: metricName
+      prefix: tinyproxy_
+
+  tags:
+    - instrumentation.name
+
+  dashboardTemplates: ./dashboard.json
+
+  compositeMetrics:
+    goldenMetrics:
```

Comment:
This is not supported now, since they are defined in another file, so the validation will fail here as well.

Reply:
@IreneMLC  Should I split summary and golden metrics to separate files?

Reply:
Yes please, and then they can be referenced like we do [here](https://github.com/newrelic-experimental/entity-synthesis-definitions/blob/main/docs/example-entity-definition.yml). Thanks!